test/metabase/pulse/render/test_util.clj

```diff
@@ -100,90 +95,18 @@
      (= tag :img)
      (str/starts-with? src "<svg"))))
 
-(defn- wrapped-children?
```

Comment:
A lot of the stuff in this namespace was unused or became unused once I deleted other unused stuff so I removed it. Removed code is debugged code right